honeybee_radiance/writer.py

```diff
@@ -238,141 +239,79 @@ def model_to_rad_folder(model, folder=None, minimal=False):
-                        mod_combs, mod_names, False, minimal)
-    
-    # write nonopaque static indoor shades
-    nos_mods, nos_mods_blk, mod_combs, mod_names = \
-        _collect_modifiers(nonopaque_shades, nonopaque_shades_blk, False)
-    _write_static_files(folder, model_folder.STATIC_NONOPAQUE_INDOOR, model_id,
-                        nonopaque_shades, nonopaque_shades_blk, nos_mods, nos_mods_blk,
-                        mod_combs, mod_names, False, minimal)
-
-    # gather static outdoor shades
-    opaque_shades, opaque_shades_blk, nonopaque_shades, nonopaque_shades_blk = \
-        model.properties.radiance.outdoor_shades_by_opaque()
-
-    # write opaque static outdoor shades
-    os_mods, os_mods_blk, mod_combs, mod_names = \
-        _collect_modifiers(opaque_shades, opaque_shades_blk, True)
-    _write_static_files(folder, model_folder.STATIC_OPAQUE_OUTDOOR, model_id,
-                        opaque_shades, opaque_shades_blk, os_mods, os_mods_blk,
-                        mod_combs, mod_names, False, minimal)
-
-    # write nonopaque static outdoor shades
-    nos_mods, nos_mods_blk, mod_combs, mod_names = \
-        _collect_modifiers(nonopaque_shades, nonopaque_shades_blk, False)
-    _write_static_files(folder, model_folder.STATIC_NONOPAQUE_OUTDOOR, model_id,
-                        nonopaque_shades, nonopaque_shades_blk, nos_mods, nos_mods_blk,
-                        mod_combs, mod_names, False, minimal)
+    model_folder.write(folder_type=folder_type, cfg=folder_config.full, overwrite=True)
+
+    # gather and write static apertures to the folder
+    aps, aps_blk = model.properties.radiance.subfaces_by_blk()
+    mods, mods_blk, mod_combs, mod_names = _collect_modifiers(aps, aps_blk)
+    _write_static_files(
+        folder, model_folder.aperture_folder(full=True), model_id,
+        aps, aps_blk, mods, mods_blk, mod_combs, mod_names, False, minimal)
+
+    # gather and write static faces
+    faces, faces_blk = model.properties.radiance.faces_by_blk()
+    f_mods, f_mods_blk, mod_combs, mod_names = _collect_modifiers(faces, faces_blk)
+    _write_static_files(
+        folder, model_folder.scene_folder(full=True), '{}_envelope'.format(model_id),
+        faces, faces_blk, f_mods, f_mods_blk, mod_combs, mod_names, True, minimal)
+
+    # gather and write static shades
+    shades, shades_blk = model.properties.radiance.shades_by_blk()
+    s_mods, s_mods_blk, mod_combs, mod_names = _collect_modifiers(shades, shades_blk)
+    _write_static_files(
+        folder, model_folder.scene_folder(full=True), '{}_shades'.format(model_id),
+        shades, shades_blk, s_mods, s_mods_blk, mod_combs, mod_names, False, minimal)
 
     # write dynamic sub-face groups (apertures and doors)
     ext_dict = {}
+    out_subfolder = model_folder.aperture_group_folder(full=True)
     for group in model.properties.radiance.dynamic_subface_groups:
         if group.is_indoor:
```

Comment:
Should I change the folder to also use `indoor` instead of interior? Right now we are using `indoor` for `dynamic scene` and `interior` for apertures.

Reply:
I think the difference in naming convention is ok because, for an aperture, we are talking about a part of the room envelope, which can be wither "exterior" envelope or "interior" envelope. For indoor shades, we are talking about things that are wholly inside the room envelope and not "interior" walls, which still go outside the "indoor" folder since they are a part of the room envelope.